cvat/apps/annotation/mask_instance.py

```diff
@@ -0,0 +1,79 @@
+# Copyright (C) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+format_spec = {
+    "name": "MASK INSTANCE",
+    "dumpers": [
+        {
+            "display_name": "{name} {format} {version}",
+            "format": "ZIP",
+            "version": "1.0",
+            "handler": "dump"
+        },
+    ],
+    "loaders": [
+    ],
+}
+
+def dump(file_object, annotations):
+    from zipfile import ZipFile
+    import numpy as np
+    import os
+    from pycocotools import mask as maskUtils
+    import matplotlib.image
+    import io
+    from collections import OrderedDict
+
+    # RGB format, (0, 0, 0) used for background
+    def genearte_pascal_colormap(size=256):
+        colormap = np.zeros((size, 3), dtype=int)
+        ind = np.arange(size, dtype=int)
+
+        for shift in reversed(range(8)):
+            for channel in range(3):
+                colormap[:, channel] |= ((ind >> channel) & 1) << shift
+            ind >>= 3
+
+        return colormap
+
+    def convert_box_to_polygon(points):
+        xtl = shape.points[0]
+        ytl = shape.points[1]
+        xbr = shape.points[2]
+        ybr = shape.points[3]
+
+        return [xtl, ytl, xbr, ytl, xbr, ybr, xtl, ybr]
+
+    colormap = genearte_pascal_colormap()
+    instance_colors = OrderedDict((idx, colormap[idx]) for idx in range(len(colormap)))
+
+    with ZipFile(file_object, "w") as output_zip:
```

Comment:
Adding some compression may be not extraneous, check `compression` c-tor parameter, e.g. `deflate`.

Reply:
@zhiltsov-max  by default, the ZipFile use ZIP_STORED so it won't use any compression.

Reply:
Yes, so I suggest adding it, or, at least, checking if it worth, as masks are `png` images, which do take some space.

Reply:
I added the ZIP_STORED in the new commit

Reply:
Please check if `ZIP_DEFLATED` helps to reduce archive size, and add this parameter.